src/test/resources/JavaclientTest/1.0.0/USER/messages_de.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
What's the purpose of adding these json files?

Reply:
This is for test multiple component. Originally there is only one component.

Reply:
Isn't translation got from singleton service? So when will these local files be used?

Reply:
When setting up Singleton service. This is the place maintaining resources.